### PR TITLE
Github, ORCID and EGI preparation for CI2023

### PR DESCRIPTION
**Pull request by annefou**

### CODE REVIEW

File: challenge/preparation/setup.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+## Register and setup to Cloud EGI JupyterHub
+
+To facilitate the reproducibility of the papers, CI2023 reproducibility challenge provides you with a ready-to-use JupyterHub with 3 flavors: 
+- Pangeo Notebook;
+- Machine Learning Pangeo nobteook;
```
Comment (sebastian-luna-valero) [inline challenge/preparation/setup.md:35]:
```suggestion
- Machine Learning Pangeo notebook;
```

File: challenge/preparation/setup.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+With all the 3 flavors, you can also add additional packages you may need for reproducing the paper of interest.
+
+For the Pangeo and Machine Learning notebook flavors, a Dask-gateway is available for scaling your computation. For more details on this deployment, you may want to take a look at [Daskhub helm chart](https://github.com/dask/helm-chart/tree/main/daskhub).
+
+
+From there, you can open and execute any notebooks used during the workshop, enjoy 🚀!
+
+
+:::{note}
+After the CI2023 reproducibility challenge, you can continue to use the resources to finalize the Jupyter Notebook to be published in the [EDS Book](https://edsbook.org), but note that we would probably modify the service and things might not work from time to time. The link (web address) to the Cloud EGI JupyterHub may change too. **Therefore, we recommend you to copy all the necxessary files before; for instance in the Github repository set up for reproducing your paper**.
```
Comment (sebastian-luna-valero) [inline challenge/preparation/setup.md:84]:
```suggestion
After the CI2023 reproducibility challenge, you can continue to use the resources to finalize the Jupyter Notebook to be published in the [EDS Book](https://edsbook.org), but note that we would probably modify the service and things might not work from time to time. The link (web address) to the Cloud EGI JupyterHub may change too. **Therefore, we recommend you to copy all the necessary files before; for instance in the Github repository set up for reproducing your paper**.
```

**Review by sebastian-luna-valero — COMMENTED** [review]
Thanks, Anne.

Find below my suggestions.

File: challenge/preparation/setup.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+### Registration
+
+
+First, as early as possible (ideally before the start of the CI2023 reproducibility challenge), all the participants will have to create an EGI account and enroll to the the Virtual Organisations to have access to Cloud EGI JupyterHub.
+
+There are several steps to follow:
+
+1. Sign up for an EGI account on the **development instance** of EGI Check-In, clicking on: [https://aai-dev.egi.eu/registry/co_petitions/start/coef:2](https://aai-dev.egi.eu/registry/co_petitions/start/coef:2). **Use your [ORCID iD](https://orcid.org/) account to authenticate.**
+2. **Enroll** in the “vo.pangeo.eu” Virtual Organisation by clicking on: [https://aai-dev.egi.eu/registry/co_petitions/start/coef:290](https://aai-dev.egi.eu/registry/co_petitions/start/coef:290). Review and click on “Submit”. Once your enrollment has been validated you will have access to [https://repro-challenge.vm.fedcloud.eu/](https://repro-challenge.vm.fedcloud.eu/) (it requires you to authenticate using EGI Check-in credentials e.g. your ORCID iD).
```
[the remark this answers is not in the document]
Comment (annefou) [reply challenge/preparation/setup.md:52]: Great! I was not sure about the enrolment message! Thanks.

**Comment by acocac** [comment]
LGTM, not sure if we'll have time to deploy and test local dockerised environments, but let's keep TBD for this item.

**Pull request merged**